### DISCUSSION TITLE
Streaming logs?

### DISCUSSION THREAD

**Discussion by classabbyamp** [buildbot/buildbot#8069] [Q&A]
Is there any way to stream logs in buildbot 4? in buildbot 0.8, we were able to use `curl -sL "http://build.voidlinux.org/builders/${arch}_builder/builds/-1/steps/shell_3/logs/stdio/text"` to get the text of the latest build's logs directly

**Comment by Charlesnorris509** [comment]
Hey @classabbyamp  there are been some changes made on the buildbot project but it still provides a REST API that allows you to query builds, steps, and logs. For more info check the link below. https://docs.buildbot.net/latest/developer/rest.html?spm=5aebb161.2ef5001f.0.0.14b05171J2cfBg

**Reply by classabbyamp** [reply]
the REST API does not *stream* logs

**Comment by p12tic** [comment]
Sorry for a delay in responding. Yes, there is no way to stream using something simple as curl. Could you please open a feature request?

**Reply by classabbyamp** [reply]
done: #8334
